src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -172,33 +172,69 @@ then you have to download the database driver and put it at: *PROACTIVE_HOME/add
 
 Scheduler, and 3 microservices (*scheduling-api*, *job-planner*, and *proactive-cloud-watch*) have to point to the same database.
 
-== Installation on a Cluster
+== Deploy ProActive Nodes
 
-Adding Compute Hosts of a cluster to the ProActive Scheduler typically involves unpacking the release archive on all those hosts.
-Once it's done you need to run a ProActive Node on the Compute Host and connect it to the ProActive Scheduler. There are two principal ways of
-doing that:
+In the ProActive Scheduler, all the computational resources (<<_glossary_compute_host,Compute Hosts>>) are managed as a set of <<_glossary_node_source,*Node Sources*>>.
+Each node source contains a set of <<_glossary_proactive_node,*ProActive Nodes*>> which shares the same deployment mechanism and access policy.
+[NOTE]
+====
+Examples of a Node Source:
+
+* a cluster with SSH access where nodes are available from 9 a.m. to 9 p.m.
+* nodes from Amazon EC2 available permanently for users from group 'cloud'.
+====
+
+image::rm-portal.png[align=center]
+As shown in the previous screenshot, the ProActive Scheduler provides a <<_glossary_rm_web_interface,*Resource Manager Web Interface*>> to show and manage all the node sources with its nodes.
+
+Adding a cluster of Compute Hosts to the ProActive Scheduler requires to *create a node source*.
+Its typical process involves copying the necessary ProActive binary files to all those Compute Hosts,
+running ProActive Nodes on the hosts, and connecting the nodes to the ProActive Scheduler.
+There are two principal ways of doing that:
+
+- Launch a process of ProActive Node *on the Compute Host* and connect it to the ProActive Scheduler
+- Initiate the deployment *from the ProActive Scheduler*
+
+If you are not familiar with ProActive Scheduler you may want to try the <<run_node_manually,*first method*>> as it's directly executed on the Compute Hosts.
```

Comment:
with ProActive server configuration,

Reply:
Do you mean "familiar with ProActive Scheduler" -> "familiar with ProActive server configuration,"?
I think the second method (deploy nodes from the ProActive Scheduler) doesn't require the users to configure the ProActive Server. Did I misunderstand something?

Reply:
familiar with ProActive scheduler is misleading, but I agree that familiar with ProActive server configuration is misleading also.
I suggest:
If you are not familiar with ProActive Workflows & Scheduling, ..

Reply:
Agree. I'll apply it.

Reply:
I change it to "If you are not familiar with ProActive Workflows & Scheduling product".

Reply:
I like it